frontend/stylesheets/main.less

```
@@ -647,3 +647,17 @@ button.narrow {
   text-align: center;
   display: inherit;
 }
+
+.segments-stats-info-alert {
+  margin-bottom: 10px;
+  margin-top: -14px; // wanted margin - modal padding = 10-24
+}
+
+.organization-switch-menu > ul {
+  max-height: 75vh;
```

This rule is not specific enough to be applied.
```suggestion
  max-height: 75vh !important;
```

Maybe you could also use `calc` like this?
```suggestion
  max-height: calc(100vh - var(--navbar-height) - 200px) !important;
```